runtime/decimal.c

```diff
@@ -275,8 +263,7 @@ static bool decimalListContains(const DecimalConstPtr *start, const DecimalConst
     initContext(&cx);
     while (start < end) {
         const DecimalConstPtr *mid = start + (end - start)/2;
-        decQuad midVal;
-        decQuadFromString(&midVal, *mid, &cx);
+        decQuad midVal = **((decQuad**)mid);
         decQuad cmp;
         decQuadCompare(&cmp, dq, &midVal, &cx);
```

Comment:
Get rid of midVal and use `(const decQuad *)mid` instead of `&midVal`.

Reply:
fixed